rubin_jupyter_utils/hub/scanrepo/scanrepo.py

```diff
@@ -655,3 +630,51 @@ def _convert_time(self, ts):
             # Naive time
             f = "%Y-%m-%dT%H:%M:%S.%f"
         return datetime.datetime.strptime(ts, f)
+
```

Comment:
Just making sure: you've tested that this can delete images both in private repositories that actually use the Docker Repo API, and from Docker Hub?  The magic was, IIRC, finding out how to actually remove an image from Docker Hub, which is going to remain important.

Reply:
No I haven't tested deleting images.  We shouldn't be deleting images this way anyway.  And we're going to get rid of it.  We shouldn't be getting rid of any private repo images right now

Reply:
....ok, but we still want to delete our public images, like we've been doing.